### PR TITLE
[DAT-19146] WalMart POC :: Informix :: Test :: Determine status of Informix integration functionality

### PR DESCRIPTION
**Pull request by Tamelianovych**
1. 
_//not expected_
Change Type 'pro:addCheckConstraint' is not allowed for Informix Dynamic Server.
Change Type 'pro:createFunction' is not allowed for Informix Dynamic Server.
Change Type 'pro:createTrigger' is not allowed for Informix Dynamic Server.
_//expected_
Change Type 'pro:createPackage' is not allowed without a valid Liquibase Pro License.
Change Type 'pro:createPackageBody' is not allowed without a valid Liquibase Pro License.

2. 
```
<changeSet author="oleh" id="1">
        <addDefaultValue  tableName="posts"
                          columnName="inserted_date"
                          columnDataType="date"
                          defaultValueComputed="DATETIME YEAR TO FRACTION(5) DEFAULT CURRENT YEAR TO FRACTION(5)"/>
    </changeSet>
```

Does not work:
Actual: 
ALTER TABLE testdb:informix.posts
    MODIFY (inserted_date date DEFAULT DATETIME YEAR TO FRACTION(5) DEFAULT CURRENT YEAR TO FRACTION(5));
Expected:
ALTER TABLE testdb:informix.posts
    MODIFY (inserted_date DATETIME YEAR TO FRACTION(5) DEFAULT CURRENT YEAR TO FRACTION(5));

  3. Snapshots does not output UniqueConstraint

  4. Cannot add valid procedure into test-harness framework

### CODE REVIEW

File: src/test/resources/harness-config.yml [hunk 1]
```diff
@@ -2,6 +2,13 @@ inputFormat: xml
 context: testContext
 
 databasesUnderTest:
+  - name: informix
```
Comment (rberezen) [inline src/test/resources/harness-config.yml:5]: duplicated entry, we have 14 and 12 at the end of the file

File: src/test/resources/docker/docker-compose.yml [hunk 2]
```diff
@@ -368,7 +368,8 @@ services:
     restart: always
     privileged: true
     volumes:
-      - "./informix-init.sql:/opt/ibm/config/informix-init.sql"
```
[the remark this answers is not in the document]
Comment (Tamelianovych) [reply src/test/resources/docker/docker-compose.yml:372]: Has this as test change, thanks for catching that

File: src/test/resources/harness-config.yml (hunk 1 above)
Comment (Tamelianovych) [reply src/test/resources/harness-config.yml]: Nice catch!

**Review by rberezen — APPROVED** [review]

**Comment by rberezen** [comment]
wow, a huge amount of work here, great job @Tamelianovych!

**Pull request merged**